src/main/java/net/io/kino/service/impl/ReservationServiceImpl.java

```diff
@@ -25,10 +35,63 @@
 
     @Override
     public Order createOrder(List<Ticket> tickets, PersonalDetails client) {
-        Order order = new Order(tickets, client, OrderState.inProgress);
+        Order order = new Order(tickets, client, OrderState.inProgress, LocalDateTime.now());
         return orders.save(order);
     }
 
+    @Override
+    public HttpResponse<com.paypal.orders.Order> createTransaction(Order order) {
+        OrdersCreateRequest request = new OrdersCreateRequest();
+        request.prefer("return=representation");
+        request.requestBody(buildRequestBody(order));
+        HttpResponse<com.paypal.orders.Order> response = null;
+        try {
+            response = client().execute(request);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        assert response != null;
```

Comment:
Ten assert nigdy nie będzie wykonany, bo domyślnie są wyłączone przez JVMke. Poza tym chyba niepotrzebne takie rzeczy w kodzie produkcyjnym.

Reply:
Wielki mamy kod produkcyjny. A tak ogólnie to się zostawia, bo możesz chcieć testowo odpalić